integration_tests/src/main/python/orc_write_test.py

```diff
@@ -357,6 +357,8 @@ def test_orc_write_column_name_with_dots(spark_tmp_path):
 @ignore_order
 def test_orc_do_not_lowercase_columns(spark_tmp_path):
     data_path = spark_tmp_path + "/ORC_DATA"
+    expected_error_message = "No StructField named acol" if is_before_spark_400() else \
+                             "Key `acol` is not exists."
```

Comment:
Is this the actual error message from Spark 4.0 now? Since the grammar seems weird maybe link to the source/relevant Spark PR?

Reply:
I'm afraid it is.  Which means we'll probably have to revisit this after the clumsy grammar is fixed in Apache Spark.

Reply:
I'll update the change with a link to the source text.